Godeps/Godeps.json

```diff
@@ -2917,73 +2917,78 @@
 		},
 		{
 			"ImportPath": "google.golang.org/grpc",
-			"Comment": "v1.3.0",
-			"Rev": "d2e1b51f33ff8c5e4a15560ff049d200e83726c5"
+			"Comment": "v1.6.0",
+			"Rev": "f92cdcd7dcdc69e81b2d7b338479a19a8723cfa3"
 		},
 		{
 			"ImportPath": "google.golang.org/grpc/codes",
-			"Comment": "v1.3.0",
-			"Rev": "d2e1b51f33ff8c5e4a15560ff049d200e83726c5"
+			"Comment": "v1.6.0",
+			"Rev": "f92cdcd7dcdc69e81b2d7b338479a19a8723cfa3"
+		},
+		{
+			"ImportPath": "google.golang.org/grpc/connectivity",
+			"Comment": "v1.6.0",
+			"Rev": "f92cdcd7dcdc69e81b2d7b338479a19a8723cfa3"
 		},
 		{
 			"ImportPath": "google.golang.org/grpc/credentials",
-			"Comment": "v1.3.0",
-			"Rev": "d2e1b51f33ff8c5e4a15560ff049d200e83726c5"
+			"Comment": "v1.6.0",
+			"Rev": "f92cdcd7dcdc69e81b2d7b338479a19a8723cfa3"
 		},
 		{
-			"ImportPath": "google.golang.org/grpc/grpclb/grpc_lb_v1",
-			"Comment": "v1.3.0",
-			"Rev": "d2e1b51f33ff8c5e4a15560ff049d200e83726c5"
+			"ImportPath": "google.golang.org/grpc/grpclb/grpc_lb_v1/messages",
```

Comment:
@dixudx do you know why the ImportPath changed?

Reply:
@caesarxuchao It was grpc/grpc-go#1466 that made this changed.